app/presenters/concerns/assetted_view.rb

```diff
@@ -36,4 +38,36 @@ def js_files
       }
     ]
   end
+
+  protected
+
+  def feature_toggle_js_var(js_var, env_var)
+    {
+      name: js_var,
+      value: feature_toggle_enabled?(env_var),
+      unquoted: true
+    }
+  end
+
+  # Checks environment to detect whether a feature is toggled on
+  #
+  # A feature is toggled on if its environment variable is set, and not "0" or
+  # "false".
+  #
+  # @param env_var [String] environment variable name
+  # TODO: move this somewhere usable by any section of the code
```

Comment:
We could extend ENV for feature toggle detection:
```ruby
# in an initializer:
def ENV.on?(name)
  key?(name) && !%w(0 false).include?(self[name])
end
def ENV.off?(name)
  !on?(name)
end
```

Reply:
I don't think this is necessary at this point where features are just set enabled/disabled via essentially a boolean env var.
If the logic for determining what is enabled got more complex or if we would need to check for enabled features in more places, I'd say we just create a global helper for feature_toggle_enabled?